Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN wget https://github.com/ethereum/solidity/releases/download/v0.8.13/solc-sta
 RUN mkdir -p /usr/app
 WORKDIR /usr/app
 
+ENV NODE_OPTIONS=--max-old-space-size=4096 
```

Comment:
Why do we increase heap limit here? Is this error caused by the changes of the Event argument type?

Reply:
This error is not related to changes in this PR and affects [other PRs](https://github.com/VenusProtocol/isolated-pools/pull/64) as well.

I am not clear on the original of the error but it is caused by javascript running out of allocated memory when compiling the contracts with hardhat